doc/neps/nep-0048-spending-project-funds.rst

```diff
@@ -0,0 +1,421 @@
+candidate within the NumPy team, or whether we look for new people to get
+involved. Before making any decisions, the decision makers should think about
+whether an opportunity should be advertised to give a wider group of people a
+chance to throw their hat in the ring for it.
+
+Compensating fairly
+```````````````````
+
+Paying people fairly is a difficult topic. Therefore, we will only offer some
+guidance here. Final decisions will always have to be considered and approved
+by the group of people that bears this responsibility (according to the
+current NumPy governance structure, this would be the NumPy Steering Council).
+
+Discussions on employee compensation tend to be dominated by two narratives:
+"pay local market rates" and "same work -- same pay".
+
+We consider them both extreme:
+
+- "Same work -- same pay" is unfair to people living in locations with a higher
+  cost of living. For example, the average rent for a single family apartment
+  can differ by a large factor (from a few hundred dollar to thousands of
+  dollars per month).
+- "Pay local market rates" bakes in existing inequalities between countries
+  and makes fixed-cost items like a development machine or a holiday trip
+  abroad relatively harder to afford in locations where market rates are lower.
+
+We seek to find a middle ground between these two extremes.
+
+Useful points of reference include companies like GitLab and
+Buffer who are transparent about their remuneration policies ([3]_, [4]_),
+Google Summer of Code stipends ([5]_), other open source projects that manage
+their budget in a transparent manner (e.g., Babel and Webpack on Open
+Collective ([6]_, [7]_)), and standard salary comparison sites.
+
+Since NumPy is a not-for-profit project, we also looked to the nonprofit sector
+for guidelines on remuneration policies and compensation levels. Our findings
+show that most smaller non-profits tend to pay a median salary/wage. We
+recognize merit in this approach: applying candidates are likely to have a
+genuine interest in open source, rather than to be motivated purely by
+financial incentives.
+
+Considering all of the above, we will use the following guidelines for
+determining compensation:
+
+1. Aim to compensate people appropriately, up to a level that's expected for
+   senior engineers or other professionals as applicable.
+2. Establish a compensation cap of $125,000 USD that cannot be exceeded even
+   for the residents from the most expensive/competitive locations ([#f-pay]_).
+3. For equivalent work and seniority,  a pay differential between locations
+   should never be more than 2x.
+   For example, if we pay $110,000 USD to a senior-level developer from New
+   York, for equivalent work a senior-level developer from South-East Asia
+   should be paid at least $55,000 USD. To compare locations, we will use
+   `Numbeo Cost of Living calculator <https://www.numbeo.com/cost-of-living/>`__
+   (or its equivalent).
```

Comment:
Sorry if it is clarified somewhere below--given two approximately equivalently-skilled candidates working in locations that might justify paying one candidate substantially less than the other, will the lower-burden candidate have preference on the basis of being able to perform x % more work for the available funds?

That is kind of like dumping the outsourcing debate on this NEP, but the the cost of living matter is indeed discussed here in some detail so it is hard to avoid thinking that this might be an ethical dilemma of sorts (it is addressed by the fully-remote i.e., GitLab policies?).

Reply:
@tylerjereddy I understand your concerns. Personally, I encourage the NumPy Steering Council, the body that is currently in charge of hiring decisions, to be “location-blind” in a selection of contractors and choose the best candidate for the job/task. After all, NumPy is a not-for-profit project.

Reply:
Location is one factor that determines how contractors spend their money. Factors like supporting family could be considered even more important than location as the contractors' families may vary a lot in size and nature. That said, I think the location and other similar factors should be discarded when negotiating the contract terms. To get the job done well and quickly defines the efficiency of spending project funds, not how contractors are likely to spend their pays - it is nobody's business, imho.

Reply:
@tylerjereddy @pearu Thank you for taking part in this very important discussion. Would you mind copying your comments to the NumPy mailing list? According to the NumPy governance document (https://numpy.org/doc/stable/dev/governance/governance.html), this type of discussion should take place there. 
You are welcome to leave comments here, on GitHub, if you catch a typo or have minor suggestions for improving the text of this document.

Reply:
https://mail.python.org/pipermail/numpy-discussion/2021-February/081531.html

Reply:
Given the amount of discussion and that we don't have experience with doing this yet, I added:

<img width="534" alt="image" src="https://user-images.githubusercontent.com/98330/109399473-50c83580-7943-11eb-9f08-188b9a754c71.png">